tests/migrations/test_commands.py

```diff
@@ -2082,6 +2082,18 @@ def test_squashed_name_without_start_migration_name(self):
             squashed_migration_file = os.path.join(migration_dir, '0001_%s.py' % squashed_name)
             self.assertTrue(os.path.exists(squashed_migration_file))
 
+    def test_squashed_name_collision(self):
+        """Using --squashed-name shouldn’t overwrite existing migrations"""
+        initial_migration_name = 'initial'
+        with self.temporary_migration_module(module='migrations.test_migrations'):
+            expected_regex = '0001_initial.*already exists'
+            with self.assertRaisesRegex(CommandError, expected_regex):
+                call_command(
+                    'squashmigrations', 'migrations', '0001', '0002',
+                    squashed_name=initial_migration_name, interactive=False,
+                    verbosity=0,
+                )
```

Comment:
There is no need to use `assertRaisesRegex()`:
```suggestion
        msg = 'Migration 0001_initial already exists. Use a different name.'
        with self.temporary_migration_module(module='migrations.test_migrations'):
            with self.assertRaisesMessage(CommandError, msg):
                call_command(
                    'squashmigrations', 'migrations', '0001', '0002',
                    squashed_name='initial', interactive=False, verbosity=0,
                )
```